frontend/app/package.json

```diff
@@ -98,15 +98,15 @@
     "stylelint-config-recommended-scss": "4.3.0",
     "stylelint-order": "4.1.0",
     "stylelint-scss": "3.21.0",
-    "ts-loader": "9.2.8",
-    "typescript": "4.7.4",
-    "unplugin-auto-import": "0.11.2",
-    "unplugin-vue-components": "0.22.7",
+    "ts-loader": "9.4.1",
```

Comment:
I have been wondering if we do actively use ts-loader anywhere or if it is a leftover from the webpack times.